### ISSUE TITLE
falsche zeitzone (oder zeit?) in timestamp

### ISSUE THREAD

**Issue by ThomasWaldmann** [cccs/cccs-website#103]
Was mir grad beim Ändern des Texts für die Cryptoparties auffiel:

Im Header wird ein UTC-DateTime-Stamp angegeben (Z == Zulu == UTC == +00:00).

Weiter unten sieht man dann aber an dem Hinweis, dass mit 11:00 wohl eher Localtime gemeint ist, was im November bei uns `+01:00` ist (also UTC + 1h).

https://en.wikipedia.org/wiki/ISO_8601

```
---
kind: event
startdate: 2019-11-16T11:00:00Z
...
---
ACHTUNG: Beginn schon um 11 Uhr und parallel zum RepairCafe.
```

**Comment by Skyr** [comment]
Hintergrund ist eine der Unzulänglichkeiten von yaml: yaml macht zwar "ein bisschen" ISO 8601, unterscheidet sich aber in diesem Detail: ISO8601 sagt, dass Zeitangaben ohne Zeitzonenangabe als local time zu interpretieren sind, yaml macht daraus aber Zulu time...

Also eigentlich meinen wir immer local time, können es aber nicht notieren...

**Comment by ThomasWaldmann** [comment]
Man koennte dann aber auch `+01:00` (bzw. `+02:00` DST) explizit angeben, dann waere es korrekt, oder?

**Comment by Skyr** [comment]
Ja, aber dann müsste der Autor jedes Eintrags wissen, ob gerade Sommerzeit herrscht. Und den Skripten, die die regelmäßigen Termine für ein Jahr im Voraus generieren, müsste man das auch beibringen...

**Comment by ThomasWaldmann** [comment]
Jo. Naja, vielleicht werden wir die Umschaltung ja bald los, geplant isses ja.

**Comment by Skyr** [comment]
Eigentlich hätte ich gern eine Notation im Sinne von "an Datum X zur Uhrzeit Y in Region Z"...

**Comment by ThomasWaldmann** [comment]
https://github.com/toml-lang/toml#local-date-time just digged the place where toml specifies behaviour (after remembering that toml fixes some issues of other formats).
